src/Unosquare.Swan/Networking/JsonClient.cs

```diff
@@ -134,7 +135,7 @@ public class JsonClient
                 var response = await httpClient.PostAsync(url, payloadJson, ct);
 
                 if (response.IsSuccessStatusCode == false)
-                    throw new Exception($"Error POST Json. Status code: {response.StatusCode}");
+                    throw new JsonRequestException("Error POST JSON", (int) response.StatusCode);
 
                 return await response.Content.ReadAsStringAsync();
```

Comment:
not sure if there is a need to await here..you can just return the ReadAsStringAsync result to the caller.